### PR TITLE
Pet Species

### PR DESCRIPTION
**Pull request by Itswali**
Update The Table.

`- Insertion of data `

`-  Species Part`

`- Addition or deletion `

`-  Weight Part `
 

`-  Question Part `

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -5,3 +5,13 @@ VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23),
        (2, 'Gabumon', '2018-11-15', 2, true, 8),
        (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
        (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+--  FOR 2nd Part
+
+INSERT INTO animals VALUES (1,' Charmander', '2020-02-08', 0, false, 11);
+INSERT INTO animals VALUES (2,' Plantmon', '2021-11-15', 2, true, 5.7);
+INSERT INTO animals VALUES (3,' Squirtle', '1993-04-02', 3, false, 12.13);
+INSERT INTO animals VALUES (4,' Angemon', '2005-06-12', 1, true, 45);
```
Comment (BenMukebo) [inline data.sql]: - [ ] Great job so far, 👏🏻  However, you need to pay more attention to the project requirements, actually the fourth first insert should have `weight_kg` with a negative value. kindly correct this 👍🏻  Your code should look like this:

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');
```

File: queries.sql [hunk 2]
```diff
@@ -23,3 +23,31 @@ WHERE name != 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+--  FOR PART 2
```
Comment (BenMukebo) [inline queries.sql:28]: - [ ] Great job so far 👏🏻 However, you are missing some important queries you should start with👇🏻 Please add them and be sure to take a screenshot of the results of your actions. 📸


image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Itswali,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow was correctly used.  ✔️ 
- Professional README file. ✔️  
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Itswali** [comment]
Updated Image for Queries of Species.
Screenshot from 2023-03-01 14-52-03

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Itswali👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Well done with query results. ✔️
- Professional README file. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**